httplib.h

```diff
@@ -192,8 +192,11 @@ using ssize_t = long;
 #include <winsock2.h>
 #include <ws2tcpip.h>
 
+#if __has_include(<afunix.h>)
```

Comment:
@p1-0tr this is not compatible with C++11.

Reply:
I know it only got standardised as of c++17, but has been supported by all major compilers before that. I've added an additional check that should protect against errors on compilers that do not support that macro.